source/coll_db.f90

```diff
@@ -506,6 +732,29 @@ subroutine cdb_writeDB_HDF5
 end subroutine cdb_writeDB_HDF5
 #endif
 
+! ================================================================================================ !
+!  V.K. Berglyd Olsen, BE-ABP-HSS
+!  Created: 2019-09-02
+!  Updated: 2019-09-02
+!  Rotate a collimater by rAngle radians
+!  Note: does not check that zero <= rAngle <= twopi
+! ================================================================================================ !
+subroutine cdb_rotateCollimator(collID, rAngle)
+
+  use numerical_constants
+
+  integer,          intent(in) :: collID
+  real(kind=fPrec), intent(in) :: rAngle
+
+  cdb_cRotation(collID) = cdb_cRotation(collID) + rAngle
+  if(cdb_cRotation(collID) > twopi) then
+    cdb_cRotation(collID) = cdb_cRotation(collID) - twopi
+  else if(cdb_cRotation(collID) < zero) then
+    cdb_cRotation(collID) = cdb_cRotation(collID) + twopi
```

Comment:
why not:
```
tmpAngle = cdb_cRotation(collID) / twopi
cdb_cRotation(collID) = ( tmpAngle + int(tmpAngle) ) * twopi
```

Reply:
This does not work. Should be `tAngle - floor(tAngle)`